index.js

```diff
@@ -123,7 +123,7 @@ class LatticeKeyring extends EventEmitter {
   // Add addresses to the local store and return the full result
   async addAccounts(n=1) {
     if (n === CLOSE_CODE) {
```

Comment:
You can remove this entire block. It's a remnant from a previous design

Reply:
Separately, you can also remove this block as it is quite old and no longer necessary: https://github.com/GridPlus/eth-lattice-keyring/blob/master/index.js#L93

Reply:
removed both 9fb5aa5